### ISSUE TITLE
Events in Custom View can't be trigger.

### ISSUE THREAD

**Issue by amcuserguy** [andreamazz/AMPopTip#98]
UIButton's Touch Up Inside event can't triggered when the Custom View is being presented. Any thoughts?

**Comment by andreamazz** [comment]
Hi @amcuserguy 
You need to disable the dismiss on tap.

**Comment by amcuserguy** [comment]
@andreamazz still not working :(

**Comment by andreamazz** [comment]
@amcuserguy 
I just tested it in the sample project and it works, can you post the code that you are using to add the button?

**Comment by amcuserguy** [comment]
My button is on ActionViewController. all the events are set on the said controller.

ActionViewController *actionView = [[ActionViewController alloc] init];
        [actionView.view addSubview:actionView._benefitsView];
        actionView.view.frame = actionView._benefitsView.frame;

        self.popTip.shouldDismissOnTap = YES;
        
        [self.popTip showCustomView:actionView.view direction:AMPopTipDirectionDown inView:self.view fromFrame:_profileView.frame];

**Comment by andreamazz** [comment]
```
    self.popTip.shouldDismissOnTap = NO;
```

**Comment by andreamazz** [comment]
@amcuserguy 
did it work?

**Comment by andreamazz** [comment]
Closing this, feel free to reopen if needed.

**Comment by rivera-ernesto** [comment]
I have the same problem, the tip disables my collection view cell's segue even when setting:

```        tip.shouldDismissOnTap = false
        tip.shouldDismissOnTapOutside = false
        tip.shouldDismissOnSwipeOutside = false
```

**Comment by rivera-ernesto** [comment]
Found the problem. You add gesture recognizers to the container view (my collection view) which interfere with taps on my cells.

```
self.containerView?.addGestureRecognizer(self.tapRemoveGestureRecognizer ?? UITapGestureRecognizer())
      self.containerView?.addGestureRecognizer(self.swipeGestureRecognizer ?? UITapGestureRecognizer())
```

**Comment by bharat-madept** [comment]
I have the same problem in the latest version (2.1.6). Here is my code 
let popTip = PopTip()
        popTip.padding = 0.0
        popTip.cornerRadius = 12
        popTip.borderWidth = 0.0
        popTip.bubbleColor = UIColor.white
        popTip.arrowSize = CGSize(width: 28, height: 19)
        popTip.shouldDismissOnTap = false
        popTip.shouldDismissOnTapOutside = false
        popTip.shouldDismissOnSwipeOutside = false
By setting tap and swipe to false doesn't make any difference.

**Comment by andreamazz** [comment]
I see. Where are you presenting the poptip from?

**Comment by incidiuz** [comment]
Hi, i have same problem.
i added table as costum view but i cant triggered when i click that table.
i have set shouldDismissOnTap to false.

thanks.

**Comment by andreamazz** [comment]
Should be fixed in #143
